### DISCUSSION TITLE
osu![lazer] crashed unexpectedly while loading up a beatmap

### DISCUSSION THREAD

**Discussion by Modsnotworking** [ppy/osu#16695] [Q&A] [answered]
I loaded a beatmap then the game freezed for 3-4 seconds and then it crashed. 
Logs:
[logs.zip](https://github.com/ppy/osu/files/7962924/logs.zip)

**Comment by peppy** [comment, marked as answer]
What kind of crash are you seeing? The logs don't show the game crashing at all. That said, there is a mention of your video decode failing due to hardware acceleration, so maybe try turning that off (search "hardware accel" in settings).

**Reply by peppy** [reply]
Please try disabling hardware acceleration and see if it works after doing so.

**Reply by Modsnotworking** [reply]
> Please try disabling hardware acceleration and see if it works after doing so.

ok I will use that setting
